examples/gfl-local.html

```diff
@@ -162,7 +162,14 @@
         }
       ],
       translations: {
-        Edition_license: 'Editionslizenz'
+        de: {
+          Edition_license: 'Editionslizenz',
+          next_page : "Nächste Seite",
```

Comment:
Why did you add these keys here? TIDO should only have next_item, previous_item, next_manuscript, previous_manuscript. Every project should only configure those 4 keys.